go/engine/merkle_audit.go

```diff
@@ -27,7 +27,7 @@ var (
 var MerkleAuditSettings = BackgroundTaskSettings{
 	Start:        5 * time.Minute,
 	StartStagger: 1 * time.Hour,
-	Interval:     6 * time.Hour,
+	Interval:     24 * time.Hour,
```

Comment:
Just wondering, do you know how this works on mobile? Is it 24 hours real time or amount of time in foreground?

Reply:
Good Q, I don't know... @pzduniak ?

Reply:
Seems to only run on wifi

Reply:
I think josh did that for the box auditor too a while back.

Reply:
>do you know how this works on mobile?
there should be an early return if it detects its not running on desktop, unless someone changed it

Reply:
ok I asked miles, he says timer is real time